tests/post/steps/test_network.py

```diff
@@ -0,0 +1,108 @@
+import json
+
+import pytest
+from pytest_bdd import given, scenario, then
+
+from tests import utils
+
+
+@scenario("../features/network.feature", "All expected listening processes")
+def test_all_listening_processes(host):
+    pass
+
+
+@given("we run on an untainted single node")
+def running_on_single_node_untainted(k8s_client):
+    nodes = k8s_client.list_node()
+
+    if len(nodes.items) != 1:
+        pytest.skip("We skip single node cluster for this test")
```

Comment:
I think you meant "skip multi nodes clusters" :wink:

Reply:
:smiley: right